test/definitions/apm/php-agent/php-apache-fpm-wordpress-linux2.json

```diff
@@ -0,0 +1,46 @@
+{
+    "global_tags": {
+        "owning_team": "OpenSource",
+        "Environment": "development",
+        "Department": "Product",
+        "Product": "Virtuoso"
+    },
+
+    "resources": [{
+      "id": "php-a-f-w-l2",
+      "display_name": "PHP Apache FPM Wordpress",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3a.micro",
+      "ami_name": "amazonlinux-2-base*",
+      "user_name": "ec2-user"
+    }],
+
+    "services": [
+        {
+          "id": "wordpress",
+          "display_name": "RedHat PHP Apache FPM Wordpress",
+          "local_source_path": "/mnt/open-install-library",
+          "deploy_script_path": "test/deploy/linux/php/apache-fpm-wordpress/redhat/roles",
+          "port": 80,
+          "destinations": ["php-a-f-w-l2"]
+        }
+    ],
+
+    "instrumentations": {
+      "resources": [
+        {
+          "id": "instrumentation1",
+          "resource_ids": ["php-a-f-w-l2"],
+          "provider": "newrelic",
+          "local_source_path": "/mnt/open-install-library",
+          "deploy_script_path": "test/deploy/linux/newrelic-cli/install-recipe/roles",
+          "params": {
+            "recipe_content_url": [
+              "https://raw.githubusercontent.com/newrelic/open-install-library/feat/php-agent/recipes/newrelic/apm/php/redhat.yml"
```

Comment:
This should be updated to `https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/apm/php/redhat.yml`
The unit tests are updating the main to the feature branch name when running on a PR

Reply:
@zsistla I believe this still needs to be updated, maybe an oversight?